WalletWasabi.Fluent/ViewModels/SearchBar/SearchItems/TransactionSearchItem.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections.Generic;
+using System.Linq;
+using NBitcoin;
+using WalletWasabi.Fluent.ViewModels.Wallets.Home.History.HistoryItems;
+
+namespace WalletWasabi.Fluent.ViewModels.SearchBar.SearchItems;
+
+internal class TransactionSearchItem
```

Comment:
Is this class needed for this PR?

Reply:
No. It's a left over from the transaction search feature, that was removed from this PR. 
Fixed.
Thanks for noticing!